CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Mediagrains Library Changelog
 
+## 2.8.1 (Under Construction)
+- Added a bypass so that when wrapping a BytesIO in an Async wrapper unnecessart and potentially costly threads aren't spawned
```

Comment:
Minor spelling, `unnecessarily`, but it would be good to have this fix on master (which you were probably planning to do anyway).